index.js

```diff
@@ -14,11 +14,11 @@ function migrate(persistence, config, action, migrationPath, migration, callback
                     sequelize.Sequelize
                 ]
             }
-        });
+        }).on('migrating', (name) => console.log('Running -->', name));
```

Comment:
For backwards compatibility can this be changed to use a standard function rather than an arrow function?